File: plugins/fr.cea.nabla.ui/examples/DaceExamples/src/simplemeshexample/quad.py
'''
Created on 10 déc. 2021

@author: meynardr
'''

class Quad:
    '''
    classdocs
    '''


    def __init__(self, id1, id2, id3, id4):
        '''
        Constructor
        '''
        self.__id1 = id1
        self.__id2 = id2
        self.__id3 = id3
        self.__id4 = id4
        
    def __str__(self):
        return [self.__id1, self.__id2, self.__id3, self.__id4]

File: plugins/fr.cea.nabla.javalib/src/fr/cea/nabla/javalib/mesh/Quad.java
/**
 * Copyright (c) 2022 CEA
 * This program and the accompanying materials are made available under the
 * terms of the Eclipse Public License 2.0 which is available at
 * http://www.eclipse.org/legal/epl-2.0.
 * 
 * SPDX-License-Identifier: EPL-2.0
 * Contributors: see AUTHORS file
 */
package fr.cea.nabla.javalib.mesh;

public class Quad extends NodeIdContainer
{
	public Quad( int id1,  int id2,  int id3,  int id4)
	{
		super(new int[] { id1, id2, id3, id4 });
	}
}
